src/connectors/multi-index-results/connectMultiIndexResults.js

```diff
@@ -0,0 +1,112 @@
+import find from 'lodash/find';
+
+import escapeHits, { tagConfig } from '../../lib/escape-highlight';
+import { checkRendering } from '../../lib/utils';
+
+const usage = `Usage:
+var customMultiIndexResults = connectMultiIndexResults(function render(params, isFirstRendering) {
+  // params = {
+  //   derivedIndices,
+  //   instantSearchInstance,
+  //   widgetParams
+  // }
+});
+search.addWidget(
+  customMultiIndexResults({
+    indices,
+    [ escapeHits = false ]
+  })
+);
+Full documentation available at https://community.algolia.com/instantsearch.js/connectors/connectMultiIndexResults.html
+`;
+
+export default function connectMultiIndexResults(renderFn, unmountFn) {
+  checkRendering(renderFn, usage);
+
+  return (widgetParams = {}) => {
+    const { indices } = widgetParams;
+
+    if (
+      !indices ||
+      !Array.isArray(indices) ||
+      (Array.isArray(indices) && indices.length < 1)
```

Comment:
Not sure about the last condition, we can still render even if we have only one index (the default one). By taking this path we can also add a default value for `indices` and avoid the first condition (expect if we still want to avoid the `null` case).